fastlane_bot/helpers/routehandler.py

```diff
@@ -501,13 +502,13 @@ def get_custom_address(
             return pool.tkn0_address
 
 
-    def generate_flashloan_struct(self, trade_instructions_objects: List[TradeInstruction]) -> List:
+    def generate_flashloan_struct(self, trade_instructions_objects: List[TradeInstruction]) -> Tuple[List, int]:
```

Comment:
This function does nothing but calling `self._get_flashloan_struct`, so you may as well just get rid of it and rename the other one to `generate_flashloan_struct`.

Reply:
Yes, I am aware of it, but it is unrelated to the changes in the PR.